sklearn_hierarchical/classifier.py

```diff
@@ -512,7 +541,7 @@ def _make_base_estimator(self, node_id):
         return LogisticRegression()
 
     def _progress(self, total, desc, **kwargs):
-        if self.interactive:
-            return tqdm_notebook(total=total, desc=desc)
+        if self.progress_wrapper:
```

Comment:
To enable progress bars in both notebook & embedded environments, switched to a user-driven generator